eventstore/migrations/0022_cdu_address_update.py

```diff
@@ -0,0 +1,72 @@
+# Generated by Django 2.2.8 on 2020-04-06 08:15
+
+import uuid
+
+import django.utils.timezone
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("eventstore", "0021_covid19triage")]
+
+    operations = [
+        migrations.CreateModel(
+            name="CDUAddressUpdate",
+            fields=[
+                (
+                    "id",
+                    models.UUIDField(
+                        default=uuid.uuid4,
+                        editable=False,
+                        primary_key=True,
+                        serialize=False,
+                    ),
+                ),
+                (
+                    "created_by",
+                    models.CharField(blank=True, default="", max_length=255),
+                ),
+                ("timestamp", models.DateTimeField(default=django.utils.timezone.now)),
+                ("first_name", models.CharField(max_length=255)),
+                ("last_name", models.CharField(max_length=255)),
+                (
+                    "id_type",
+                    models.CharField(
+                        choices=[
+                            ("sa_id", "SA ID"),
+                            ("passport", "Passport"),
+                            ("dob", "Date of birth"),
+                        ],
+                        max_length=8,
+                    ),
+                ),
+                ("id_number", models.CharField(blank=True, max_length=13, null=True)),
+                ("date_of_birth", models.DateField(blank=True, null=True)),
+                ("folder_number", models.CharField(max_length=255)),
+                ("municipality", models.CharField(max_length=255)),
+                ("city", models.CharField(max_length=255)),
+                ("suburb", models.CharField(max_length=255)),
+                ("street_name", models.CharField(max_length=255)),
+                ("street_number", models.CharField(max_length=255)),
+            ],
+        ),
+        migrations.AlterField(
```

Comment:
Seems to be from previous PR

Reply:
Hmm that's weird.. I wonder if we need to `sort` this field, so that it's in the same order every time.